src/providers/twitch/TwitchAccount.cpp

```diff
@@ -444,4 +445,35 @@ void TwitchAccount::autoModDeny(const QString msgID, ChannelPtr channel)
         });
 }
 
+const QString &TwitchAccount::getSeventvUserID() const
+{
+    return this->seventvUserID_;
+}
+
+void TwitchAccount::loadSeventvUserID()
+{
+    if (!this->seventvUserID_.isEmpty() || this->isAnon())
+    {
+        return;
+    }
+
+    static const QString seventvUserInfoUrl =
+        QStringLiteral("https://7tv.io/v3/users/twitch/%1");
+
+    NetworkRequest(seventvUserInfoUrl.arg(this->getUserId()),
```

Comment:
Should probably be part of a seventvapi file since we already implement this API in SeventvEmotes.cpp